interfaces/builtin/dcdbas_control.go

```diff
@@ -45,13 +45,30 @@ const dcdbasControlConnectedPlugAppArmor = `
 /sys/devices/platform/dcdbas/host_control_action rw,
 /sys/devices/platform/dcdbas/host_control_smi_type rw,
 /sys/devices/platform/dcdbas/host_control_on_shutdown rw,
+
+# additional devices required by dchbas library
+/dev/EsmBASDev r,
+/dev/port r,
+/dev/mem r,
```

Comment:
This makes this super privileged and makes me thinking if we shouldn't have a separate interface like 'memory-observe' for such memory access. @jdstrand what do you think about this?

Furthermore dcdbas is specific to Dell devices but an official kernel interface as described in https://www.kernel.org/doc/Documentation/dcdbas.txt so I am not sure if its correct to merge library implementation specifics in here.